2-mandatory/1-writers.js

```diff
@@ -61,6 +61,17 @@ Exercise 1:
 */
 function logAllWriters() {
   // write your code to log all writers here
+for (let item of writers){
+
+let firstName = item[0];
+let lastName = item[1];
+let age = item[3];
+let occupation =item[2];
+}
+console.log(
+  "Hi, my name is {firstName} {lastName}. I am {age} years old, and work as a {occupation}."
+);
```

Comment:
I think this console.log isn't quite in the right place. Perhaps have a look at how many times it's being called